openssl-dynamic/src/main/c/sslcontext.c

```diff
@@ -40,6 +40,8 @@
 
 extern apr_pool_t *tcn_global_pool;
 
+static char *requestedSignature;
```

Comment:
consider clarifying this variable name to provide more context (e.g. `keyMaterialRequestedMethodSignature`)